lib/optimize.js

```diff
@@ -132,7 +133,7 @@ async function optimizeByType({ fileBuffer, filePath, isLossless }) {
         ? execBuffer({
           bin: guetzli,
           args: ['--quality', '90', execBuffer.input, execBuffer.output],
-          input: fileBuffer,
+          input: await setSrgbColorspace(fileBuffer),
```

Comment:
Not sure that it's fine to set color space when user does not expect that. But let's see how it will work 🤔